README.md

```diff
@@ -33,4 +33,62 @@ The number of vowels and consonants of screen_name and name is counted, and it i
+
+#### Implementation
+
+The heuristic consists of evaluating the interactions (RTs, Quotes, Mentions, etc.) generated by the user being analyzed and doing some mathematical computations about them.
+Since performing the preparations for this computations for every user may imply doing an extra computation for every user that she/he started an interaction with, and since not all interacted users may be stored in the document for each user, an upper limit is set on the number of the interacted users that are going to be taken into account for the computations (NUM_INTERACTED_USERS). It can be set in the 'bot_detector.py' file. Note that it's not the same constant than that of the 'fake_promoter.py' file, and it cannot be greater than the maximum number of interacted users stored in the db for each user. When describing the heuristic, this detail is ignored for simplicity purposes, but this must be taken into account when executing the script.
+
+For illustrating how the heuristic works, let's take for example the user CESARSANCHEZ533.
+
+Assume that he's the user being analyzed.
+First, the interactions and users interacted by him are fetched, using the 'NetworkAnalyzer' class from the 'network_analysis' module(3). The results are better if the synthesis of the interactions of a user is ordered decreasingly by the number of interactions started with them, so its iterated in this order.
+Also the synthesis of the interacted users with the bot_detector_pbb of each one of them is fetched.
+The exact next steps depends on the specific method of the heuristic, and are described below. For now suffice it say that a score is calculated, and if it trespassess a Threshold, then the heuristic returns 1. If not, it returns 0.
+
+There are two approaches and four different methods for computing the value of the heuristic -the score-. All of them implemented, and one can switch between one and another by setting the constant FAKE_PROMOTER_HEUR (in the method 'compute_bot_probability') to the number corresponding to the desired heuristic.
+
+##### Approach Number 1
+
+The main idea of this approach is to compute the number of interactions with users with a bot_detector_pbb higher than some minimum value (BOT_DET_PBB_THRS), and what fraction of the total number of interactions it represents.
+
+###### Method 0
+
+'Filter' the accounts by its bot_detector_pbb, selecting only those that are greater than a minimum (BOT_DET_PBB_THRS).
+Compute the total number of interactions started by the analyzed user (in the example, CESARSANCHEZ533) with the filtered users.
+Evaluate if this number is greater than some threshold(4) (SCORE_TOP_INTRCTNS_THRESHOLD), and return 1 if it is. Otherwise return 0.
+Then divide over the total number of interactions of the user to get the percentage of the total that these interactions represents, and evaluate them against another threshold, and do the same as before.
```

Comment:
Eliminar el *then*. La explicación de este último párrafo hay que mejorar. Es muy vaga, por ejemplo a que threshold nos referimos?. Cuando decimos *do the same as before* a que nos referimos concretament. Además, no entiendo a que porcentaje nos referimos. Si te complica explicar en inglés podes poner en español y después yo traduzco.